mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/AndroidManifest.xml

```diff
@@ -11,7 +11,8 @@
   <uses-feature android:name="android.hardware.camera" />
   <!-- For using the internet (GIF acquisition) -->
   <uses-permission android:name="android.permission.INTERNET" />
-  
+  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
Are we writing to storage?

Reply:
Original Tenor GIF Keyboard required this, but my new changes should make it unnecessary. Also removed the need for internet permissions as well.